gfxr_ext/decode/dive_annotation_processor.cpp

```diff
@@ -61,9 +61,18 @@ void DiveAnnotationProcessor::WriteBlockEnd(const gfxrecon::util::DiveFunctionDa
             if (vkCmd.name == "vkBeginCommandBuffer")
             {
                 m_cmd_vk_commands_cache[cmd_handle].clear();
+                m_cmd_buffer_draw_call_counts_map[cmd_handle].push_back(0);
             }
 
             m_cmd_vk_commands_cache[cmd_handle].push_back(vkCmd);
+
+            if (function_name.find("vkCmdDraw") != std::string::npos)
```

Comment:
there are more vk events that we need consider as drawcall:
vkCmdDrawIndexed
vkCmdDrawIndirect
vkCmdDrawIndexedIndirect
vkCmdDrawIndirectCount
vkCmdDrawIndexedIndirectCount